connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/ExecuteHolder.scala

```diff
@@ -85,8 +106,19 @@ private[connect] class ExecuteHolder(
   /**
    * Interrupt the execution. Interrupts the running thread, which cancels all running Spark Jobs
    * and makes the execution throw an OPERATION_CANCELED error.
+   * @return
+   *   true if it was not interrupted before, false if it was already interrupted.
    */
-  def interrupt(): Unit = {
+  def interrupt(): Boolean = {
     runner.interrupt()
   }
+
+  /**
+   * Spark Connect tags are also added as SparkContext job tags, but to make the tag unique, they
+   * need to be combined with userId and sessionId.
+   */
+  def tagToSparkJobTag(tag: String): String = {
```

Comment:
@juliuszsompolski input `tag` isn't used for output, which doesn't look intended

Reply:
sure